packages/nimble-components/src/radio-group/tests/radio-group.stories.ts

```diff
@@ -75,7 +75,7 @@ const metadata: Meta<RadioGroupArgs> = {
         },
         name: {
             description:
-                'Radio buttons whose values are mutually exclusive should set the same `name` attribute. Setting the name on the group sets it on all child radio buttons.'
+                'Radio buttons whose values are mutually exclusive should set the same `name` attribute. Setting the name on the group sets it on all child radio buttons. When using radio buttons in an Angular form, you must explicitly set either `name` or `formControlName` on each radio button rather than only setting `name` on the group.'
```

Comment:
I considered creating an `.mdx` and putting this in a separate Angular usage section, but I thought it would be harder to miss here in the property doc.